### PR TITLE
Update associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl

### PR DESCRIPTION
**Pull request by PriscilaSJUFRGS**
Atualização do estilo

### CODE REVIEW

File: associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl [hunk 1]
```diff
@@ -82,7 +83,7 @@
     <choose>
       <if type="chapter entry-dictionary entry-encyclopedia">
         <names variable="container-author" delimiter=", ">
-          <name name-as-sort-order="all" sort-separator=", " initialize-with=". " delimiter="; " delimiter-precedes-last="always">
+          <name name-as-sort-order="all" sort-separator=", " delimiter="; " delimiter-precedes-last="always">
```
Comment (adam3smith) [inline associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl:86]: is the removal of `initialize-with` here intentional? It's used everywhere else in the style

Comment (PriscilaSJUFRGS) [reply associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl:86]: Thanks, @adam3smith! 
It was an error. Can you fix it for us or must I submmit another pull request?

Comment (adam3smith) [reply associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl:86]: I can fix

**Comment by adam3smith** [comment]
Obrigado

**Pull request merged**